hooks/useVehicles.ts

```diff
@@ -21,7 +21,7 @@ export const useVehicles = () => {
     }
   }
 
-  const defaultVehicle = vehicles.length > 0 ? vehicles[0] : null
+  const defaultVehicle = vehicles?.[0] || null
```

Comment:
Use `??` to prevent `undefined`. Or if it's okay to have `undefined` as vehicle, `??/|| null` does not make sense at all.

Reply:
undefined is prevented as you can see in the screenshot with `||` syntax as well
<img width="182" alt="Screenshot 2023-11-23 at 14 01 09" src="https://github.com/bratislava/paas-mpa/assets/61841194/1e18c4ca-5050-416e-9a84-97e865fb2a41">

Reply:
yes, sorry, you are correct